package.json

```diff
@@ -25,31 +25,30 @@
 		"url": "https://github.com/gisat/ptr-locales"
 	},
 	"dependencies": {
-		"i18next": "^19.8.7",
-		"lodash": "^4.17.20",
+		"i18next": "^20.3.2",
+		"lodash": "^4.17.21",
 		"react": "16.13.1",
-		"react-i18next": "9.0.10"
+		"react-i18next": "^11.11.0"
 	},
 	"devDependencies": {
-		"@babel/cli": "^7.12.16",
-		"@babel/core": "^7.12.16",
-		"@babel/plugin-proposal-class-properties": "^7.12.13",
-		"@babel/preset-env": "^7.12.16",
-		"@babel/preset-react": "^7.12.13",
+		"@babel/cli": "^7.14.5",
+		"@babel/core": "^7.14.6",
+		"@babel/preset-env": "^7.14.5",
+		"@babel/preset-react": "^7.14.5",
 		"@gisatcz/prettier-config": "0.0.3",
-		"@rollup/plugin-commonjs": "^17.1.0",
+		"@rollup/plugin-commonjs": "^19.0.0",
 		"@rollup/plugin-json": "^4.1.0",
-		"auto": "^9.43.1",
+		"auto": "^10.29.3",
 		"babel-plugin-lodash": "^3.3.4",
 		"cross-env": "^7.0.3",
-		"husky": "^5.0.9",
+		"husky": "^6.0.0",
 		"npm-run-all": "^4.1.5",
-		"prettier": "^2.2.1",
-		"pretty-quick": "^3.1.0",
+		"prettier": "^2.3.1",
+		"pretty-quick": "^3.1.1",
 		"rimraf": "^3.0.2",
-		"rollup": "^2.39.0",
+		"rollup": "^2.52.1",
 		"rollup-plugin-babel": "^4.4.0",
-		"rollup-plugin-filesize": "^9.1.0",
+		"rollup-plugin-filesize": "^9.1.1",
```

Comment:
migrate to @rollup/plugin-babel

Reply:
fixed: 7644deab652196c9bd02aea9c101c3e5206dced2